blog/2024-03-01-config-entry-reconfigure-step.md

```diff
@@ -0,0 +1,44 @@
+---
+author: G Johansson
+authorURL: https://github.com/gjohansson-ST
+title: "Config Entries can now provide a reconfigure step"
+---
+
+As of Home Assistant Core 2024.4 integrations can add a `reconfigure` step in their config entries to allow a user to reconfigure necessary config entry data.
```

Comment:
It's the config flow which has a reconfigure step:
```suggestion
As of Home Assistant Core 2024.4 integrations can add a `reconfigure` step in their config flows to allow a user to reconfigure necessary config entry data.
```

We could also rephrase it to say config entries can be recconfigured.